packages/nhost-js/src/core/nhost-client.ts

```diff
@@ -18,7 +39,7 @@ export class NhostClient {
    * Nhost Client
    *
    * @example
-   * const nhost = new NhostClient({ url });
+   * const nhost = new NhostClient({ backendUrl: 'https://my-app.nhost.run' });
```

Comment:
Could you please wrap this example in a code block? This way our DocGen tool will be able to visualize the example for this constructor correctly.

Reply:
:) 
done